src/Illuminate/Queue/Middleware/PreventOverlappingJobs.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Illuminate\Queue\Middleware;
+
+use Illuminate\Contracts\Cache\Repository as Cache;
+
+class PreventOverlappingJobs
+{
+    /**
+     * The amount of time (in seconds) to expire the lock.
+     *
+     * @var int
+     */
+    public $expiresAt;
+
+    /**
+     * The key of the job.
+     *
+     * @var string
+     */
+    public $key;
```

Comment:
Order of these properties is odd.

Reply:
Current order is alphabetic. I thought that's what Laravel uses?